src/sentry/static/sentry/app/views/projectInstall/platform.jsx

```diff
@@ -142,13 +142,13 @@ const ProjectInstallPlatform = React.createClass({
             <div dangerouslySetInnerHTML={{__html: this.state.html}}/>
           )}
 
-          {queryParams.onboarding ?
+          {'signup' in queryParams ?
             <p>
-              <Link
-                to={`/${orgId}/${projectId}/?onboarding=1#welcome`}
+              <a
```

Comment:
this is a nice-to-have. Getting the signup stuff out is more important to me in the short term, so I can just change this back to <Link>. Or @macqueen if you have any ideas on how to propagate the state change, happy to do that.